tests/testsuite/git.rs

```diff
@@ -323,6 +323,20 @@ fn cargo_compile_with_malformed_nested_paths() {
             "#,
             )
             .file("vendor/dep2/Cargo.toml", "!INVALID!")
+            .file(
+                "vendor/dep3/Cargo.toml",
+                r#"
+            [project]
+            name = "dep3"
+            version = "0.5.0"
+
+            [dependencies]
+            subdep1 = { path = "../require-extra-build-step" }"#,
```

Comment:
Can you clean up the indentation here?  Maybe something like this?
```suggestion
                r#"
                [project]
                name = "dep3"
                version = "0.5.0"
                
                [dependencies]
                subdep1 = { path = "../require-extra-build-step" }
                "#,
```